plugins/LocalContainerProvider/LocalContainerProvider.py

```diff
@@ -288,7 +302,9 @@ def _pathToMime(self, path) -> Optional[MimeType]:
         return mime
 
     ##  Converts a file path to the ID of the container it represents.
-    def _pathToId(self, path) -> Optional[str]:
+    def _pathToId(self, path: str) -> Optional[str]:
+        result = None
         mime = self._pathToMime(path)
         if mime:
-            return urllib.parse.unquote_plus(self._pathToMime(path).stripExtension(os.path.basename(path)))
+            result = urllib.parse.unquote_plus(mime.stripExtension(os.path.basename(path)))
+        return result
```

Comment:
Can be simplified to:

```python
if mime:
    return urllib.parse.unquote_plus(mime.stripExtension(os.path.basename(path)))
return None
```

Nice to make fewer variables right? Especially if they're gonna get mutated.

Reply:
This only has one `return` in this function which is easier to read IMO.

Reply:
ok, fair enough